pkg/bundle/bundle.go

```diff
@@ -18,6 +18,7 @@ import (
 
 	"github.com/go-git/go-git/v5/plumbing/format/gitignore"
 	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
no need for the `v1` qualifier with the package name ends in v1

surprised govet or goimports did not catch this

either drop the v1 import qualifier, or change to something more meaniful like containerregv1

Reply:
or `containerreg` is fine too, as I just noticed the pattern you employed in the new results.go file